src/helpers/notifications/notifications.ts

```diff
@@ -0,0 +1,7 @@
+import Toast from 'react-native-simple-toast';
+
+function notify(text: string) {
+  Toast.show(text, Toast.LONG);
```

Comment:
I think we need at least a few of them (error, warning(?), info, success)

Reply:
Sorry, maybe I was not very clear. We will need some notifications that can appear on top of the screen and have different styles for info / error kinds of cases